jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java

```diff
@@ -40,44 +103,57 @@ public void setLength(double length) {
         this.length = length;
     }
 
+    /**
+     * Retrieves the speedup factor applied by the layer for this action.
+     *
+     * @see Action#setSpeed(double) for detailed documentation
+     * @return the speed of frames.
+     */
     public double getSpeed() {
         return speed;
     }
 
+    /**
+     * Alters the speedup factor applied by the layer running this action.
+     * This factor controls the animation direction, so if the speed is a positive value then the animation would run forward and vice versa.
+     * The speed factor gets applied, inside the {@link com.jme3.anim.AnimLayer}, on each interpolation step by this formula : time += tpf * action.getSpeed() * composer.globalSpeed.
+     * Default speed is 1.0, it plays the animation clips at their normal speed.
+     *
+     * @param speed the speed of frames.
+     */
```

Comment:
I guess setting the speed to zero will stop the animation and setting it to a negative number will reverse it, @Ali-RS Should we document that too?

Reply:
Yeah, would be nice to add that I think.

Reply:
Done, check it out.